### PR TITLE
force pull fixes / debug

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees @csrwng PTAL ...

### CODE REVIEW

**Comment by gabemontero** [comment]
[testonlyextended][extended:core(forcePull)]

**Comment by bparees** [comment]
lgtm but i see the test failed, do you want to investigate that further before this gets merged?

**Comment by gabemontero** [comment]
Similar to another PR I just attempted an extended test on, this extended test failed with a vagrant / AWS hiccup:  not enough capacity in the US east zone, prior to the extended test running.

That said, I'd still like to see a pre-merge run with the changes in the PR.

File: test/extended/util/image_helpers.go [hunk 1]
```diff
@@ -30,6 +30,21 @@ func ResetImage(tags map[string]string) {
 
 }
 
+//DumpImage is a helper that inspects the image along with some ginkgo debug
+func DumpImage(name string) {
+	g.By(fmt.Sprintf("Calling docker inspect for image %s", name))
```
Comment (csrwng) [inline test/extended/util/image_helpers.go]: I would think that e2e.Logf would be more appropriate here.
g.By is used to describe test cases

Comment (csrwng) [inline test/extended/util/image_helpers.go]: or directly fmt.Fprintf() to g.GinkgoWriter

Comment (gabemontero) [inline test/extended/util/image_helpers.go]: went with the latter (even got some nice blue highlighting in emacs with it :-) ) ... thanks - will push in a moment

**Comment by gabemontero** [comment]
[testonlyextended][extended:core(forcePull)]

**Comment by gabemontero** [comment]
Somehow "forcePull" got converted to "forcepull" and the extended test failed with not being able to find the custom focus....I'll see if I can find some sort of logic like that in the vagrant plugin.

**Comment by gabemontero** [comment]
[testonlyextended][extended:core(builds)]

**Comment by gabemontero** [comment]
@bparees - i ran the full builds suite given the forcepull/forcePull conundrum (I decided not the change the "forcePull" string in the Describe to "forcepull" at this time), and the forcePull test passed.

Please hit the merge button unless you've had a change of opinion from earlier.

**Comment by bparees** [comment]
the other g.By() invocations should probably be converted as well.

**Comment by gabemontero** [comment]
other newly added g.By() invocations pushed

**Comment by gabemontero** [comment]
[testonlyextended][extended:core(builds)]

**Comment by bparees** [comment]
lgtm but needs secondary merge approval.  @smarterclayton this only affects extended tests, no product code.

**Comment by smarterclayton** [comment]
Approved

On Apr 19, 2016, at 10:30 PM, Ben Parees <notifications@github.com> wrote:

lgtm but needs secondary merge approval. @smarterclayton
<https://github.com/smarterclayton> this only affects extended tests, no
product code.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/8562#issuecomment-212217696>

**Comment by bparees** [comment]
[merge]

Ben Parees | OpenShift

**Comment by bparees** [comment]
[merge]

**Pull request merged**